wiki_articles/strict_weak.md

```diff
@@ -0,0 +1,29 @@
+# What Is a Strict Weak Ordering in C++?
+
+Algorithms such as
+**[std::sort](https://en.cppreference.com/w/cpp/algorithm/sort)** and
+**[std::set](https://en.cppreference.com/w/cpp/container/map)** require a
```

Comment:
Set isn't an algorithm - I'm not sure how to improve the wording right off hand but something should exist

Reply:
Done.